CHANGELOG.md

```diff
@@ -13,6 +13,24 @@
   - `register` and `register_from_file` on class `snowflake.snowpark.udtf.UDTFRegistration`
   - `register` and `register_from_file` on class `snowflake.snowpark.stored_procedure.StoredProcedureRegistration`
   - `udf`, `udtf` and `sproc` in `snowflake.snowpark.functions`
+- Support `Column` as an input argument to a stored procedure call
+- Support `table_type` in `save_as_table`. You can now choose from three different `table_type` options: `"temporary"`, `"temp"` and `"transient"`
+- Added the ability to only get a query ID after evaluating a `DataFrame` instead of fetching results
+
+### Improvements:
+- Validate object name in session.use_* methods
+- Added new flake8 plugin
+- Escape query tag in SQL when it has special characters
+- Check if anaconda terms are acknowledged when adding missing packages
```

Comment:
- Added a check to see if Anaconda terms are acknowledged when adding missing packages.